docs/learn/address-prefilling/upgraded-version.md

```diff
@@ -0,0 +1,467 @@
+## Upgraded version
```

Comment:
Hi @RadimFab, thank you for your contribution, and I apologize for my late reply.

We discussed this internally and decided that we would not like to publish or reference such a long code snippet to the University since we cannot guarantee its quality and maintenance. Instead, we believe it's better to publish it to your own GitHub repository and remove it from here.

That being said, I've asked our developers to include this functionally in the store extension, so it is available for everyone without the need for a fork (issue reference EXE-2454).

Thank you for your understanding.